manifests/install.pp

```diff
@@ -192,7 +192,7 @@
   # Render the Shibboleth configuration. These are run time and not used
   # during the build process. It should restart Jetty, though.
   ['ldap.properties', 'idp.properties', 'authn/general-authn.xml', 'logback.xml',
-    'c14n/subject-c14n.xml', 'saml-nameid.xml',
+    'c14n/subject-c14n.xml', 'saml-nameid.xml', 'access-control.xml',
```

Comment:
The template is not linted, so it should be fine. You could also be specific in the template like so:
https://docs.puppet.com/puppet/3.8/lang_template_erb.html#scopevariable-or-scopelookupvarvariable

https://github.com/puppetlabs/puppetlabs-mysql/blob/master/templates/my.cnf.pass.erb#L7